egret/parsers/prescient_dat_parser.py

```diff
@@ -29,14 +28,17 @@ def create_ModelData(dat_file):
     '''
     return md.ModelData(create_model_data_dict(dat_file))
 
-
 def create_model_data_dict(dat_file):
-    
-    abstract_params = AbstractModel()
+    model = get_uc_model()
+    params = model.create_instance(dat_file)
+    return create_model_data_dict_params(params)
 
-    load_basic_data(abstract_params)
+def get_uc_model():
+    uc_model = AbstractModel()
+    load_basic_data(uc_model)
```

Comment:
I would argue the "load_basic_data" function is incorrectly named - it's data free, but rather is just adding components to an AbstractModel shell. Am I missing something?

Reply:
Changed to `setup_abstract_model`. This function inherited its name from some predecessor long ago..